src/main/java/org/opendatakit/aggregate/constants/ErrorConsts.java

```diff
@@ -26,6 +26,7 @@
  */
 public final class ErrorConsts {
 
+  public static final String POSSIBLE_CORRUPTION = "Encountered an unknown problem while reading the database. This often is a sign of database corruption. Please get the server logs and file an issue";
```

Comment:
This is a catch-all error for unhandled exceptions

Reply:
How about adding https://github.com/opendatakit/aggregate/issues after add an issue?